### PR TITLE
ACS Installation part 2

### PR DESCRIPTION
**Pull request by gaurav-nelson**
- Includes instructions for installing the `central-service` Helm chart.

Preview:
- Installing quickly https://openshift-docs-git-acs-install-2-gnelson.vercel.app/openshift-acs/master/installing/installing_helm/install-helm-quick.html
- Installing with customizations https://openshift-docs-git-acs-install-2-gnelson.vercel.app/openshift-acs/master/installing/installing_helm/install-helm-customization.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bergerhoffer PTAL

@IAmATeaPot418 I've implemented the info based on your comments from https://github.com/openshift/openshift-docs/pull/32076 PTAL

File: modules/scanner-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/prerequisites.adoc
+:_module-type: CONCEPT
+[id="scanner-prerequisites_{context}"]
+= Prerequisites for installing Scanner
+
+{product-title} includes an image vulnerability scanner called _Scanner_.
```
Comment (bergerhoffer) [inline modules/scanner-requirements.adoc:8]: This is the only place in these prereq docs where you italicize the component name (Scanner). Maybe not needed?

File: modules/sensor-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/prerequisites.adoc
+:_module-type: CONCEPT
+[id="sensor-prerequisites_{context}"]
+= Prerequisites for installing Sensor
+Sensor monitors your Kubernetes and OpenShift clusters.
```
Comment (bergerhoffer) [inline modules/sensor-requirements.adoc:7]: OpenShift -> {ocp}

**Comment by gaurav-nelson** [comment]
@bergerhoffer I wasn't able to copy the images symlink into the new `installing/installing_helm` directory. Do you know how to do that? or in general how to copy modules and images symlink into all new folders?

**Comment by bergerhoffer** [comment]
> @bergerhoffer I wasn't able to copy the images symlink into the new `installing/installing_helm` directory. Do you know how to do that? or in general how to copy modules and images symlink into all new folders?

@gaurav-nelson You can just create a new symlink by running the following command inside the `installing/installing_helm` directory:

```
$ ln -s ../../images images
```
It looks like the modules symlink is actually only backing up one directory, so you should probably delete and recreate that one as well. I think it'll technically still work because it'll go into the parent directory's symlink, but would be good to have it be correct and back up two levels.

**Comment by bergerhoffer** [comment]
BTW I just merged a PR I had open that talked about these symlinks, so there are some more details about creating them in our guidelines now here: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#directory-names

File: installing/installing_helm/install-helm-quick.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* Review the xref:../prerequisites.adoc[prerequisites for installing {product-title}].
+
+include::modules/adding-helm-repository.adoc[leveloffset=+1]
+
+include::modules/acs-quick-install-using-helm.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
+
+* xref:./install-helm-customization.adoc[Installing with customizations using Helm charts].
```
Comment (bergerhoffer) [inline installing/installing_helm/install-helm-quick.adoc:23]: Probably doesn't need a period at the end

File: modules/adding-helm-repository.adoc [hunk 4]
```diff
@@ -14,12 +15,20 @@
 $ helm repo add stackrox  https://mirror.openshift.com/pub/rhacs/charts/
 ----
 +
-This Helm chart repository includes the following charts to install various components:
+The Helm repository for {product-title} includes two Helm charts for installing different components.
+
+** Central services Helm chart (`central-services`) for imstalling the centralized components (Central and Scanner).
```
Comment (bergerhoffer) [inline modules/adding-helm-repository.adoc:20]: imstalling -> installing

File: modules/central-services-public-config.adoc [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+| The port number on which to expose Central.
+The default port number is 443.
+
+| `central.exposure.nodePort.enabled`
+| Use `true` to expose Central by using the NodePort service.
+
+| `central.exposure.nodePort.port`
+| The port number on which to expose Central.
+When you skip this parameter, {ocp} automatically assigns a port number.
+Red Hat recommends that you don't specify a port number if you are exposing {product-title} by using a node port.
```
Comment (bergerhoffer) [inline modules/central-services-public-config.adoc:137]: don't -> do not

File: modules/central-services-private-config.adoc [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+| `imagePullSecrets.password`
+| The password (for the selected username) of your registry.
+
+| `imagePullSecrets.allowNone`
+| Use `true` if you are using a custom registry and it allows pulling images without credentials.
+
+| `imagePullSecrets.useExisting`
+| A comma-seprated list of secrets as values.
+For example, `+secret1, secret2, secretN`. Use this option if you have already created pre-existing image pull secrets with the given name in the target namespace.
```
Comment (bergerhoffer) [inline modules/central-services-private-config.adoc:39]: There's a "+" in front of the secret list, should that be removed?

**Review by bergerhoffer — COMMENTED** [review]
Other than the symlinks, just a few last little things!

**Review by bergerhoffer — COMMENTED** [review]
LGTM, thanks for the updates!

**Pull request merged**